apps/judicial-system/backend/migrations/20211220090749-update-case.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  up: (queryInterface) => {
+    // replaceEnum does not support transactions
+    return queryInterface.sequelize.transaction(() =>
```

Comment:
This should do the job. The transaction is redundant here, however, as it is not used.

Consider removing the transaction.

Reply:
Awesome, fixed :)